Core/src/com/serotonin/m2m2/web/mvc/spring/security/authentication/MangoTokenAuthenticationProvider.java

```diff
@@ -77,14 +76,14 @@ public Authentication authenticate(Authentication authentication) throws Authent
         userDetailsChecker.check(user);
 
         if (log.isDebugEnabled()) {
-            log.debug("Successfully authenticated user using JWT token, header: " + jws.getHeader() + ", body: " + jws.getBody());
+            log.debug(String.format("Successfully authenticated user using JWT token, header: %s, body: %s", jws.getHeader(), jws.getBody()));
```

Comment:
I would rather this change was not made, I know the IDE is probably warning you about string concatenation on this line but there are 2 reasons to not touch this:
- If we ever change this in 5.x and need to backport again there will be a conflict
- Its not the correct fix, we should use `log.debug("Successfully authenticated user using JWT token, header: {}, body: {}", jws.getHeader(), jws.getBody());` (this is actually the fix that IntelliJ suggests, see screenshot)

![image](https://github.com/MangoAutomation/ma-core-public/assets/3579645/37a28fa4-1992-4580-8f64-1eefff969073)

Reply:
ok, done

Reply:
Not done